.githooks/pre-commit

```diff
@@ -57,7 +60,10 @@ function generate_standalone_yaml() {
   # update kustomize files for OLM bundle with files generated for openshift
   cp "${charttmpdir}/mongodb-kubernetes/templates/operator.yaml" config/manager/manager.yaml
   cp "${charttmpdir}/mongodb-kubernetes/templates/database-roles.yaml" config/rbac/database-roles.yaml
-  cp "${charttmpdir}/mongodb-kubernetes/templates/operator-roles.yaml" config/rbac/operator-roles.yaml
+  cp "${charttmpdir}/mongodb-kubernetes/templates/operator-roles-base.yaml" config/rbac/operator-roles-base.yaml
+  cp "${charttmpdir}/mongodb-kubernetes/templates/operator-roles-clustermongodbroles.yaml" config/rbac/operator-roles-clustermongodbroles.yaml
+  cp "${charttmpdir}/mongodb-kubernetes/templates/operator-roles-telemetry.yaml" config/rbac/operator-roles-telemetry.yaml
+  cp "${charttmpdir}/mongodb-kubernetes/templates/operator-roles-webhook.yaml" config/rbac/operator-roles-webhook.yaml
```

Comment:
Ok, this should fail but it doesn't because we do not clean up correctly. https://github.com/mongodb/mongodb-kubernetes/pull/259 should fix the clean up.